src/server.ts

```diff
@@ -1,18 +1,37 @@
-import express from 'express'
-import { Request, Response } from 'express'
-import person from './routes/scientist'
+import express, { Request, Response } from 'express';
+import scientistsRouter from './routes/scientist';
+const morgan = require('morgan')
 
 
 var app = express();
-const port = process.env.PORT || 3000;
+
+// Config
+var port = process.env.PORT || 3000;
 const HOST = '0.0.0.0';
+if (!module.parent) { app.listen(port); }
```

Comment:
found this solution here -> https://www.marcusoft.net/2015/10/eaddrinuse-when-watching-tests-with-mocha-and-supertest.html @pedrocecchetti

Reply:
Show de bola. Eu consegui rodar os testes nao sei como, mas lembro de ter visto os logs e td bonitinho. Mas nao faco nem ideia de como fiz.